index.html

```diff
@@ -2260,6 +2260,84 @@ <h3>Role Mapping Table</h3>
+        </ul>
+        <p>Similarly, non-style generic elements are mapped similarly or identically to other generic elements when the same styles are applied. For example:</p>
+        <ul>
+          <li><code>&lt;span&gt;</code> is mapped the same as <code>&lt;div style="display: inline;"&gt;</code></li>
+          <li><code>&lt;div&gt;</code> is mapped the same as <code>&lt;span style="display: block;"&gt;</code></li>
+        </ul>
+        <div class="ednote" style="border: solid 2px red;">
+          <p>Implementors, are the previous statements true in all, or is this unique by rendering engine?</p>
+        </div>
+
+        <div class="ednote" style="border: solid 2px red;">
+          <p>The following table won't be close to accurate in the first draft PR. I'm listing these out to solicit feedback from implementors as to whether this approach is reasonable for documenting various mappings for the generic role.</p>
+        </div>
+
+        <table class="map-table elements" id="generic-role-mapping-table">
+          <caption>Platform-specific mapping characteristics of the <a class="role-reference" href="#generic"><code>generic</code></a> role.</caption>
+          <thead>
+            <tr>
+              <th><abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> Role</th>
+              <th>MSAA + IAccessible2</th>
+              <th><abbr title="User Interface Automation">UIA</abbr></th>
+              <th><abbr title="Accessibility Toolkit">ATK</abbr>/<abbr title="Assistive Technology - Service Provider Interface">AT-SPI</abbr></th>
+              <th><abbr title="macOS Accessibility Protocol">AX API</abbr><sup>[<a href="#ftn.note1">Note 1</a>]</sup></th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr id="role-map-generic-bikeshed"><!-- change 'bikeshed' here to soemthing other than CSS 'block' so as to not conflate the style property values with the layout concept -->
+              <th><a class="role-reference" href="#generic"><code>generic</code></a>, rendered as a flow contents container (E.g. <code>&lt;div&gt;</code>)</th>
+              <td class="role-msaa-ia2">
+                <span class="property">Role: <code>ROLE_SYSTEM_GROUPING</code></span><br />
+                <span class="property">Role: <code>IA2_ROLE_SECTION</code></span>
+              </td>
+              <td class="role-uia">
+                <span class="property">Control Type: <code>Group</code></span>
+              </td>
+              <td class="role-atk">
+                <span class="property">Role: <code>ROLE_SECTION</code></span><br />
+              </td>
+              <td class="role-axapi">
+                <span class="property">AXRole: <code>AXGroup</code></span><br />
+                <span class="property">AXSubrole: <code>&lt;nil&gt;</code></span><br />
+              </td>
+            </tr>
+            <tr id="role-map-generic-">
```

Comment:
Another bikeshed ID: `"role-map-generic-???"`. Commenting so I won't forget about this b/c it's been too long.